react-native-meridian-maps/android/src/main/java/com/meridianmaps/LocationSharingFragment.kt

```
@@ -0,0 +1,243 @@
+        LocationSharing.shared().uploadUserPhoto(requireContext(), uri, object : LocationSharing.Callback<User> {
+            override fun onSuccess(result: User) {
+                AlertDialog.Builder(activity!!)
+                        .setMessage("Success! image uploaded.")
+                        .setPositiveButton("OK", null)
+                        .show()
+            }
+
+            override fun onError(t: LocationSharingException) {
+                if (activity != null) {
+                    AlertDialog.Builder(activity!!)
+                            .setMessage(String.format("Error uploading image. %s", t.errorMessage))
```

detekt (1.23.8)

[warning] String.format("Error uploading image. %s", t.errorMessage) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)